sdk/samples/iot/provisioning/src/paho_iot_provisioning_sas_example.c

```diff
@@ -29,11 +29,22 @@
 #include <azure/core/az_span.h>
 #include <azure/iot/az_iot_provisioning_client.h>
 
+#define TIMEOUT_MQTT_RECEIVE_MS (60 * 1000)
+#define TIMEOUT_MQTT_DISCONNECT_MS (10 * 1000)
+
 #ifdef _MSC_VER
 // "'getenv': This function or variable may be unsafe. Consider using _dupenv_s instead."
 #pragma warning(disable : 4996)
 #endif
 
+// DO NOT MODIFY: Service information
+#define ENV_GLOBAL_PROVISIONING_ENDPOINT_DEFAULT "ssl://global.azure-devices-provisioning.net:8883"
```

Comment:
Is this something the user should never change, even if the endpoint running in a specific region or was private?

Reply:
from Hub code comment
//Paho requires that the MQTT endpoint be of the form ssl://<HUB ENDPOINT>:8883

Reply:
// Paho requires that the MQTT endpoint be of the form ssl://\<HUB ENDPOINT\>:8883

Reply:
Right, so couldn't the `<HUB ENDPOINT>` be  different based on the user's needs? Why does the  `// DO NOT MODIFY: Service information` comment apply to it?